### ISSUE TITLE
Discover face lifting 

### ISSUE THREAD

**Issue by goranjovic** [status-im/status-react#1874]
### User Story

As a user I want to be able to see important status updates, dapps and other relevant info in my Status network in simple, intuitive and visually appealing screens.

### Description

[comment]: # (Feature or Bug? i.e Type: Bug)
*Type*: Feature

Implement the subset of the following designs that matches the functionalities currently provided by the backend:

**Prepare Discover for squad development and refurnish existing features** 

- [x] Discover list item (user status)
- [x] Recent statuses screen
- [x] Popular hashtags in statuses screen
- [x] Search statuses by tags
- [x] Popular tags preview/lead snippet 
- [x] Recent statuses preview/lead snippet

All items above are collectively included in issue #1977

---------------------------
**Functionally complete work:**
- [x] #1968 DApps carousel   
- [x] #1972 all DApps list   
- [x] #1973  DApp detail screen
- [x] #1969 Public chats teaser
- [x] #1999 Images for empty recent and popular statuses widget

-------------------------------

- [x] https://github.com/status-im/status-react/issues/2221 Must haves pixel perfection

-------------------------------

The list of features intentionally left to be scoped better in the following iterations.

- Create status call to action heading
- Everything related to public chats
- Statuses posted by DApp
- Option to favorite a DApp and/or add it to contacts
- Option to chat with DApp 

- Nice to haves: https://github.com/status-im/status-react/issues/2222

**Comment by sla-shi** [comment]
Clickable prototype
https://projects.invisionapp.com/share/52DI5CFKB#/screens/253484943

**Comment by oskarth** [comment]
Issue can probably be closed once must haves #2221 is done.

**Comment by oskarth** [comment]
Fixed in develop, will be in 0.9.12 release

**Comment by yenda** [comment]
![253554100](https://user-images.githubusercontent.com/1181225/31823563-c0c5db66-b5ac-11e7-950f-1efa8854688e.png)

![screenshot_1508506467](https://user-images.githubusercontent.com/1181225/31823570-c361865e-b5ac-11e7-855c-7b08757e4d6b.png)
